SignalServiceKit.podspec

```diff
@@ -17,15 +17,37 @@ An Objective-C library for communicating with the Signal messaging service.
 
   s.homepage         = "https://github.com/WhisperSystems/SignalServiceKit"
   s.license          = 'GPLv3'
-  s.author           = { "Frederic Jacobs" => "github@fredericjacobs.com" }
+  s.author           = { "Whisper Systems" => "ios@whispersystems.com" }
   s.source           = { :git => "https://github.com/WhisperSystems/SignalServiceKit.git", :tag => s.version.to_s }
-  s.social_media_url = 'https://twitter.com/FredericJacobs'
+  s.social_media_url = 'https://twitter.com/WhipserSystems'
+
+  deprecation_message = <<EOS
```

Comment:
This is a little awkward, but it's the only way I know to print a "caveat" style message to the user after running `pod install`.

The output is like this:

```
→ pod install
Analyzing dependencies
Fetching podspec for `SignalServiceKit` from `../SignalServiceKit`
Downloading dependencies
Using 25519 (2.0.2)
Using AFNetworking (3.1.0)
...
Using libPhoneNumber-iOS (0.9.10)
Generating Pods project
Integrating client project
Sending stats
Pod installation complete! There are 7 dependencies from the Podfile and 20 total pods installed.

[!] SignalServiceKit has been deprecated in favor of installing SignalServiceKit via the Signal-iOS repository.

To get future updates, simply point your Podfile at the new location:

    pod 'SignalServiceKit', git: 'https://github.com/WhisperSystems/SignalServiceKit.git'

To this:

    pod 'SignalServiceKit', git: 'https://github.com/WhisperSystems/Signal-iOS.git'

Sorry for the disruption!

```